src/sensorpushbuttongroup.cpp

```diff
@@ -0,0 +1,118 @@
+/* antimicrox Gamepad to KB+M event mapper
+ * Copyright (C) 2022 Max Maisel <max.maisel@posteo.de>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "sensorpushbuttongroup.h"
+
+#include "buttoneditdialog.h"
+#include "inputdevice.h"
+#include "joybuttontypes/joysensorbutton.h"
+#include "joysensor.h"
+#include "joysensorbuttonpushbutton.h"
+#include "joysensoreditdialog.h"
+#include "joysensorpushbutton.h"
+
+#include <QDebug>
+#include <QHash>
+#include <QWidget>
+
+SensorPushButtonGroup::SensorPushButtonGroup(JoySensor *sensor, bool keypadUnlocked, bool displayNames, QWidget *parent)
+    : QGridLayout(parent)
+    , m_sensor(sensor)
+    , m_display_names(displayNames)
+    , m_keypad_unlocked(keypadUnlocked)
+{
+    m_left_button = generateBtnToGrid(SENSOR_LEFT, 1, 0);
+    m_right_button = generateBtnToGrid(SENSOR_RIGHT, 1, 2);
+    m_up_button = generateBtnToGrid(SENSOR_UP, 0, 1);
+    m_down_button = generateBtnToGrid(SENSOR_DOWN, 2, 1);
+    m_fwd_button = generateBtnToGrid(SENSOR_BWD, 0, 2);
```

Comment:
Why `m_fwd_button` uses `SENSOR_BWD`?

Reply:
I'll swap the variable names for FWD and BWD buttons.